react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
I used `Text` because this label should have responsive text.  Is there a nice way to opt into responsive text without using `Text`?